Origin: vendor
Subject: enable crypto-nettle feature
Forwarded: not-needed

nettle is used in the whole sequoia stack in Debian.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -59,6 +59,7 @@
 [dependencies.sequoia-openpgp]
 version = "2"
 default-features = false
+features = [ "crypto-nettle" ]
 
 [dependencies.tokio]
 version = "1.19"
